app/services/remotestorage.js

```diff
@@ -0,0 +1,57 @@
+import Ember from 'ember';
+import Space from 'hyperchannel/models/space';
+import RemoteStorage from 'npm:remotestoragejs';
+import 'npm:remotestorage-module-kosmos';
+
+export default Ember.Service.extend({
+
+  rsInstance: null,
+
+  rs: function() {
+    if (this.get('rsInstance')) { return this.get('rsInstance'); }
+
+    let rs =  new RemoteStorage(/* {logging: true} */);
+    rs.access.claim('kosmos', 'rw');
+    rs.caching.enable('/kosmos/');
+    return rs;
+  }.property('rs'),
```

Comment:
I think this is supposed to depend on `rsInstance`, not `rs`.

Also, after creating a new instance, it should be saved as `rsInstance`.